src/shogun/preprocessor/FisherLDA.h

```diff
@@ -38,6 +38,7 @@
 
 #include <shogun/features/Features.h>
 #include <shogun/labels/Labels.h>
+#include <shogun/labels/MulticlassLabels.h>
```

Comment:
not sure why this is needed?

Reply:
Could not compile differently. I am not sure why

Reply:
I have doubts here as well.... the CI never complained about this but we actually have some test for fisher LDA somewhere so it was compiled before

EDIT: Just checked, the class is used but not included. Could you forward declare it instead?